app/assets/javascripts/helpers/post_photo_uploader.es6

```diff
@@ -71,6 +76,22 @@ Diaspora.PostPhotoUploader = class {
         onError: (id, name, errorReason) => this.showMessage("error", errorReason)
       }
     });
+
+    if (this.dropZoneElementIds !== undefined && this.dropZoneElementIds instanceof Array) {
+      var dropZoneElements = this.dropZoneElementIds.map(id => document.getElementById(id))
+        .filter(el => el !== undefined && el !== null);
+      if (dropZoneElements.length > 0) {
+        var dndFineUploader = this.fineUploader;
```

Comment:
Inside of the inner function 'this' didn't resolve to the PostPhotoUploader so I was torn between doing this way or making a var self = this (or that = this) and then using that.fineUploader.  Since it was only used once in that scope I chose this.  I can instead do the self at the top and then use that if preferred.

Reply:
I've confirmed this is the behavior when I am making the rest of the changes.  Should I keep it as is or would it be preferred if I do a `var self = this` at the top and use `self.fineUploader` (I've seen that used in other JavaScript code on this project in terms of naming)?

Reply:
I thought I was having problems with the resolution of 'this' in the lambda but when making these changes I'm not seeing that problem.  I'm going to switch to using the fineUploader directly.